toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/verify/WebAppUrlVerifier.java

```diff
@@ -0,0 +1,47 @@
+package io.toast.tk.agent.ui.verify;
+
+import com.google.inject.Inject;
+import io.toast.tk.agent.config.AgentConfig;
+import io.toast.tk.agent.config.AgentConfigProvider;
+import io.toast.tk.agent.ui.utils.ConfigTesterHelper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+
+
+public class WebAppUrlVerifier implements IPropertyVerifier{
+
+    private static final Logger LOG = LogManager.getLogger(WebAppUrlVerifier.class);
+
+    AgentConfigProvider webConfigProvider;
+
+    @Inject
+    public WebAppUrlVerifier(AgentConfigProvider webConfigProvider){
+      this.webConfigProvider = webConfigProvider;
+    }
+
+    @Override
+    public boolean validate() {
+        AgentConfig agentConfig = webConfigProvider.get();
+        if(agentConfig.getProxyActivate().equals("true")) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move the "true" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1132)